README.md

```diff
@@ -1,8 +1,8 @@
-# NorthEuraLex
+# CLDF dataset derived from Dellert's et al. "NorthEuraLex (Version 0.9)" from 2020
```

Comment:
AFAIR it's supposed to be "Dellert et al.'s", no?

Reply:
Yes, please change it, @tresoldi.

@xrotwang, @SimonGreenhill, what about we include tests on all these sanity issues? It is annoying to check for these tiny things in each dataset, and I prefer to just have a standard test pasted in there.

Reply:
What would a standard test look like? `if "et al." in title and "et al.'s" not in title: ...`?

Reply:
See my proposal in some weird other thread.

```
def test_valid_title(cldf_dataset, cldf_logger):
    title = cldf_dataset.metadata_dict['dc:title']
    assert title.startswith('CLDF dataset derived from')
    author = title.split('CLDF dataset derived from')[1].split('"')[0]
    _, short_title, year_ = title.split('"')
    year = year_[6:]
    assert year.isdigit()
    assert author.endswith("'s ")
```

This wold capture the problem (!). Since it checks for the most regular conditions:

1. title has two `"`
2. title ends in "from [0-9][0-9][0-9][0-9]"
3. author ends in `'s `
4. prefix is: "CLDF dataset derived from"